src/EditorFeatures/CSharpTest/AddAccessibilityModifiers/AddAccessibilityModifiersTests.cs

```diff
@@ -177,72 +174,84 @@ namespace Test
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddAccessibilityModifiers)]
         public async Task TestAllConstructsWithOmit()
         {
-            await TestInRegularAndScriptAsync(
-@"
+            await new VerifyCS.Test
+            {
+                TestState =
+                {
+                    Sources =
+                    {
+                        @"using System;
 namespace Outer
 {
     namespace Inner1.Inner2
     {
-        internal class {|FixAllInDocument:C|}
+        internal partial class [|C|] : I
```

Comment:
Why partial?

Reply:
Compile errors were fixed in ec82406ce3f0344d95516767e042cd4856e8d8e3 to allow for a constrained review. This change addresses the CS0751 error.